qiita_db/analysis.py

```diff
@@ -986,6 +987,9 @@ def _build_biom_tables(self,
                         f, "Generated by Qiita, analysis id: %d, info: %s" % (
                             self._id, label))
 
+                # let's add the regular biom without post processing
```

Comment:
One of us should comment saying the expectation is we'll always be returning a list of n biom-files, and this first append represents the minimal number of results that need to be returned.